samples/MegaDrone/CMakeLists.txt

```diff
@@ -27,7 +27,6 @@ target_link_libraries( megadrone log oboe )
 # Make the Oboe public headers available to our app
 include_directories (${OBOE_DIR}/include)
 
-# Append compiler optimization flags to both the debug and release build variants
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Ofast")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -Ofast")
-
+# Enable optimization flags: if having problems with source level debugging,
+# disable -Ofast ( and debug ), re-enable it after done debugging.
+target_compile_options(megadrone PRIVATE -Wall -Werror -Ofast)
```

Comment:
-Ofast for release only

Reply:
first review, you said:
"I agree that -O0 should be used in normal circumstances but for the MegaDrone sample it will almost certainly glitch and hence should be left as -Ofast for all variants."
the same file but conflicting comment from you, which one you are going?

Reply:
My apologies Gerry, this comment was meant for the other samples. Here's what the optimisation flag should be for each sample: 

hello-oboe: debug -O0, release -Ofast
LiveEffect: debug -O0, release -Ofast
RhythmGame: debug -O0, release -Ofast
MegaDrone: all -Ofast